src/components/Header/Header.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoForm } from '../TodoForm';
+
+type Props = {
+  query: string,
+  todos: Todo[],
+  handleInput: (input: string) => void,
+};
+
+export const Header: React.FC<Props> = ({ query, todos, handleInput }) => {
+  const activeTodos = useMemo(() => {
```

Comment:
It looks for activeTodos only, and if it finds at least one, it returns true and applies 'active' to the class, so I'm not sure how to name it otherwise 😅